requirements.txt

```diff
@@ -18,7 +18,7 @@ nuclio-jupyter~=0.9.12
 # >=1.16.5 from pandas 1.2.1 and <1.23.0 from storey
 numpy>=1.16.5, <1.23.0
 # limiting pandas to <1.5.0 since 1.5.0 causes exception in storey on casting from ns to us
-pandas~=1.2, <1.5.0
+pandas>=1.2, <3
```

Comment:
Are we good with pandas 1.5.*?  Does storey fully incapsulates the 'datetime' 1.5.* pandas bugs and its enough?